go.mod

```diff
@@ -20,29 +21,26 @@ require (
 	github.com/onsi/gomega v1.10.1
 	github.com/opencontainers/go-digest v1.0.0 // indirect
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.5.1
-	github.com/prometheus/client_model v0.2.0
-	github.com/spf13/cobra v0.0.6
+	github.com/prometheus/client_golang v1.5.1 // indirect
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.6.2
 	go.etcd.io/etcd v0.0.0-20191023171146-3cf2f69b5738
+	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073 // indirect
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
+	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
 	google.golang.org/appengine v1.6.6 // indirect
 	google.golang.org/grpc v1.26.0
-	k8s.io/api v0.18.5
-	k8s.io/apiextensions-apiserver v0.18.5
-	k8s.io/apimachinery v0.18.5
-	k8s.io/apiserver v0.18.5
-	k8s.io/client-go v0.18.5
-	k8s.io/cluster-bootstrap v0.18.5
-	k8s.io/component-base v0.18.5
+	k8s.io/api v0.18.6
+	k8s.io/apiextensions-apiserver v0.18.6
+	k8s.io/apimachinery v0.18.6
+	k8s.io/apiserver v0.18.6
+	k8s.io/client-go v0.18.6
```

Comment:
What about those recent bugfixes that went into some `1.18.x` version that affected watching? Probably not a problem in this library, but I was just wondering if we should still align?

Reply:
I bumped controller-runtime in our fork to v0.6.3, which is using k8s v0.18.6 libraries, so I also bumped k8s dependencies to v0.18.6. See this commit message https://github.com/giantswarm/cluster-api/pull/2/commits/88cd8c1024ff714c49a6ad0d2bc902f2e1698081

Reply:
TL;DR: needed controller-runtime v0.6.2 for CAPI tests (so I don't have to modify/delete upstream code from our fork), went with v0.6.3 which is the latest. controller-runtime v0.6.3 uses k8s v0.18.6, so I bumped it to that same version for the sake of consistency :)

Reply:
Ack; All good :+1: